contracts/rbn-staking/IncentivizedVotingLockup.sol

```diff
@@ -676,12 +1006,166 @@ contract IncentivisedVotingLockup is
     return min;
   }
 
+  /**
+   * @dev Uses binarysearch to find the most recent user point history delegation preceeding block
+   * @param _addr User for which to search
+   * @param _block Find the most recent point history before this block
+   * @param _nextExpiry Next expiry of the delegation
+   */
```

Comment:
based on [getPriorVotes()](https://github.com/compound-finance/compound-protocol/blob/4a8648ec0364d24c4ecfc7d6cae254f55030d65f/contracts/Governance/Comp.sol#L189) to just get delegation / received votes